src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -243,6 +243,21 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
                             </ng-template>
                         </ngx-datatable-column>
                     }
+                    <ngx-datatable-column prop="" [sortable]="true" [minWidth]="100" [width]="100">
+                        <ng-template ngx-datatable-header-template>
+                            <span class="datatable-header-cell-wrapper d-inline-block w-100 text-center" (click)="controls.onSort('results?.[0]?.assessmentNote?.note')">
+                                <fa-icon [icon]="faComment" [ngbTooltip]="'artemisApp.assessment.assessmentNote' | artemisTranslate" />
+                                <fa-icon [icon]="controls.iconForSortPropField('results?.[0]?.assessmentNote?.note')" />
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value">
+                            @if (value.results?.[0]?.assessmentNote?.note) {
+                                <span class="w-100 text-center">
+                                    <fa-icon [ngbPopover]="(value.results?.[0]?.assessmentNote)!.note" [triggers]="'hover'" [icon]="faComment" />
```

Comment:
![screenshot](https://github.com/ls1intum/Artemis/assets/169004592/80879a6e-4c42-4d9a-8ab6-5b44397139b8)

Tooltips going up are obscured by overflow. Using `body` as the container fixes this.

```suggestion
                                    <fa-icon [ngbPopover]="(value.results?.[0]?.assessmentNote)!.note" [triggers]="'hover'" [icon]="faComment" [container]="'body'" />
```

Reply:
I know that long text are not shown very nicely, but I do not know how to improve this. Actual it looks like this:
![actual](https://github.com/ls1intum/Artemis/assets/73783062/c2e58067-5417-4391-b90e-7b08b1eaf0ae)
And with your approach it looks like this:
![with body](https://github.com/ls1intum/Artemis/assets/73783062/36b29b4c-3ee2-4702-9b9a-ce6804c33031)
Where is difference? I cannot see any. Thank you for your suggestion, but it seems that it does not help.

Reply:
The tooltip is obscured by the table header. This can only be seen if the entry is in the first row of the table.

Reply:
Okay, now I understand your issue. With a tooltip in the first line, I am able to reproduce it if I make my web browser window smaller to force the tooltip to be shown about the icon and not below.

I will apply your fix. Thank you for your contribution.

Reply:
Solved with cdb0be02fae255d4604176750d11423def75ddf8